src/validations/index.js

```diff
@@ -1,4 +1,11 @@
+var i18n = require('i18next-client');
+i18n.init({
+  lng: 'en',
+  debug: true
+});
+
 module.exports = {
+  i18n: i18n,
```

Comment:
No need to export this here—as I noted in the main PR thread, you can just `require` it directly in other modules that you need it in.